classes/class-wc-connect-paypal-ec.php

```diff
@@ -68,6 +69,40 @@ public function init() {
 			}
 		}
 
+		public function register_refund_pointer( $pointers ) {
+			$target = '.refund-actions > button:first-child';
+			$link_url = wc_gateway_ppec()->ips->get_signup_url( wc_gateway_ppec()->settings->environment );
+			$pointers[] = array(
+				'id' => 'wc_services_refund_via_ppec',
+				'target' => $target,
+				'options' => array(
+					'content' => sprintf( '<h3>%s</h3><p>%s</p>',
+						__( 'Link a PayPal account' ,'woocommerce-services' ),
+						sprintf( __( "To issue refunds via PayPal Express Checkout, you will need to <a href=\"%s\">link a PayPal account</a> with the email address that received this payment.", 'woocommerce-services' ), $link_url )
+					),
+					'position' => array( 'edge' => 'bottom', 'align' => 'top' ),
+				),
+			);
+
+			wp_add_inline_script( 'wc_services_admin_pointers', sprintf( "
+				jQuery( document ).ready( function( $ ) {
+					$( '#woocommerce-order-items' )
+						.one( 'click', 'button.refund-items', function() {
+							setTimeout( function() {
+								$( 'div.wc-order-refund-items' ).promise().then( function() {
```

Comment:
Why is a promise needed here?

Reply:
The intent is to wait for the `slideDown` animation to finish, at which point the promise ([defaulting to `fx` type](https://api.jquery.com/promise/#promise-type-target)) will resolve, and the pointer will show in the right place. It still seems to show up with a slightly incorrect vertical position on occasion — not sure why.